### PR TITLE
Remove all references of Opera [Now Flagged as Not Supported]

### PR DESCRIPTION
**Pull request by metal-messiah**
### Overview
Opera is no longer a supported browser in SauceLabs (source). While it was supported, we experienced issues with it that lead us to explicitly exclude it from certain tests in the agent (search for .exclude('opera@<=12') in the agent repo for examples), these are unnecessary now.

### Related Github Issue
[Remove Testing References](https://github.com/newrelic/newrelic-browser-agent/issues/72)
[Remove Error Stacks Parsing for Opera](https://github.com/newrelic/newrelic-browser-agent/issues/71)

### Testing
Tests do note execute against Opera, so removing references to Opera should have no consequence.  Tests should execute and succeed as before.

### CODE REVIEW

File: tests/assets/js/vendor/mootools-1.6.0.js [hunk 1]
```diff
@@ -983,7 +983,7 @@ var parse = function(ua, platform){
 	// before checking if it's chrome.
 	var UA = ua.match(/(edge)[\s\/:]([\w\d\.]+)/);
 	if (!UA){
-		UA = ua.match(/(opera|ie|firefox|chrome|trident|crios|version)[\s\/:]([\w\d\.]+)?.*?(safari|(?:rv[\s\/:]|version[\s\/:])([\w\d\.]+)|$)/) || [null, 'unknown', 0];
```
Comment (martinkuba) [inline tests/assets/js/vendor/mootools-1.6.0.js:986]: This is a third-party file that does not need to be changed.

File: tests/functional/xhr/ajax-events.test.js [hunk 2]
```diff
@@ -149,7 +149,8 @@ testDriver.test('Distributed Tracing info is added to XHR ajax events', xhrBrows
         enabled: true
       },
       ajax: {
-        harvestTimeSeconds: 2
+        harvestTimeSeconds: 2,
+        enabled: true
```
Comment (martinkuba) [inline tests/functional/xhr/ajax-events.test.js:153]: Any idea why this was not failing before?

**Review by martinkuba — CHANGES_REQUESTED** [review]
There are still a few 3rd-party JS files that should not have been changed (any file that is in the vendor folder).

File: tests/functional/xhr/ajax-events.test.js (hunk 2 above)
Comment (metal-messiah) [reply tests/functional/xhr/ajax-events.test.js:153]: i wasn't there for the merge, but i had the same question.  thought it was maybe some sort of merge conflict where the local version didn't have the flag?

File: tests/assets/js/vendor/mootools-1.6.0.js [hunk 3]
```diff
@@ -2098,6 +2098,12 @@ local.setDocument = function(document){
 				brokenSecondClassNameGEBCN = (testNode.getElementsByClassName('a').length != 2);
 			} catch (e){}
 
+			// Opera 9.6 getElementsByClassName doesnt detects the class if its not the first one
```
Comment (martinkuba) [inline tests/assets/js/vendor/mootools-1.6.0.js:2101]: This file is still showing as modified.

Comment (metal-messiah) [reply tests/assets/js/vendor/mootools-1.6.0.js]: looks like sourcetree inserted it twice when it "reverted" for some reason.  thanks for catching

**Review by martinkuba — APPROVED** [review]

**Pull request merged**